templates/includes/sidebar.html

```diff
@@ -0,0 +1,8 @@
+{% block sidebar %}
+	<ul>
+		<li><a href="#">Home page</a></li>
+		<li><a href="#">Manufacturers</a></li>
+		<li><a href="#">Cars</a></li>
+		<li><a href="#">Drivers</a></li>
+	</ul>
+{% endblock %}
```

Comment:
Incorrect Use of `{% block %}`: Since `sidebar.html` is included in another template, it should not define a `{% block sidebar %}`. Remove the `{% block sidebar %}` and `{% endblock %}` tags, and directly include the HTML content.